test-d/mutable.ts

```diff
@@ -0,0 +1,11 @@
+import {expectType} from 'tsd';
+import {Mutable} from '..';
+
+type Foo = {
+	readonly a: number;
+	readonly b: string;
+};
+
+const ab: Mutable<Foo> = {a: 1, b: '2'};
+expectType<{a: number; b: string}>(ab);
```

Comment:
Sadly, immutable object types are assignable to mutable ones. So this test is basically a no-op, it will accept mutable and immutable objects without erroring out.

You shoud change this to:
```suggestion
ab.a = 2;
```